pkg/apis/internalversion/kwokctl_configuration_types.go

```diff
@@ -210,6 +210,9 @@ type KwokctlConfigurationOptions struct {
 
 	// BindAddress is the address to bind to.
 	BindAddress string
+
+	// CertAddress is the address to generate certificate
+	CertAddress string
```

Comment:
Refer to the name of this field in Kubeadm

```suggestion
	// KubeApiserverCertSANs sets extra Subject Alternative Names for the API Server signing cert.
	KubeApiserverCertSANs []string
```

https://github.com/kubernetes/kubernetes/blob/b3056ea9bdb4d8d8e0fbb15e08dba1bfa294b813/cmd/kubeadm/app/apis/kubeadm/types.go#L160-L159